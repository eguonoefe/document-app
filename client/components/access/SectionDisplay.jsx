import React from 'react';
import { Link } from 'react-router-dom';
/**
 * Section Display component that displays a carousel.
 * @returns {String} The HTML markup for the Section Display component
 */
const SectionDisplay = () => (
  <div className="section">
    <div className="carousel carousel-slider center" data-indicators="true">
      <div className="carousel-fixed-item">
        <span className="inner-slide">
          <img
            src="../../img/slide5.jpg"
            alt="slide1" className="slide-img" />
        </span>
        <div className="landing-content">
          <h2>DocNinja</h2>
          <Link to="/signup">Get Started</Link>
        </div>
      </div>
      <div className="carousel-item white-text" href="#one!">
        <img
          src="../../img/slide6.jpg"
          alt="slide" className="slide-img" />
      </div>
      <div className="carousel-item amber white-text" href="#two!">
        <img
          src="../../img/slide1.jpg"
          alt="slide3" className="slide-img" />
      </div>
      <div className="carousel-item green white-text" href="#three!">
        <img
          src="../../img/slide4.jpg"
          alt="slide4" className="slide-img" />
      </div>
      <div className="carousel-item blue white-text" href="#four!">
        <img
          src="../../img/slide3.jpg"
          alt="slide3" className="slide-img" />
      </div>
    </div>
  </div>
);

export default SectionDisplay;
