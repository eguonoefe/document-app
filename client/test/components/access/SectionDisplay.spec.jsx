/* global expect jest test */
import React from 'react';
import { shallow, configure } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import SectionDisplay from '../../../components/access/SectionDisplay';

configure({ adapter: new Adapter() });

jest.dontMock('../../../components/access/SectionDisplay');

describe('SectionDisplay Component', () => {
  test('should match the sectiondisplay snapshot', () => {
    const component = shallow(<SectionDisplay />);
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
});
