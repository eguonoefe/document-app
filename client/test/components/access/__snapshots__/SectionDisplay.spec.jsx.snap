// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionDisplay Component should match the sectiondisplay snapshot 1`] = `
<div
  className="section"
>
  <div
    className="carousel carousel-slider center"
    data-indicators="true"
  >
    <div
      className="carousel-fixed-item"
    >
      <span
        className="inner-slide"
      >
        <img
          alt="slide1"
          className="slide-img"
          src="../../img/slide5.jpg"
        />
      </span>
      <div
        className="landing-content"
      >
        <h2>
          DocNinja
        </h2>
        <Link
          to="/signup"
        >
          Get Started
        </Link>
      </div>
    </div>
    <div
      className="carousel-item white-text"
      href="#one!"
    >
      <img
        alt="slide"
        className="slide-img"
        src="../../img/slide6.jpg"
      />
    </div>
    <div
      className="carousel-item amber white-text"
      href="#two!"
    >
      <img
        alt="slide3"
        className="slide-img"
        src="../../img/slide1.jpg"
      />
    </div>
    <div
      className="carousel-item green white-text"
      href="#three!"
    >
      <img
        alt="slide4"
        className="slide-img"
        src="../../img/slide4.jpg"
      />
    </div>
    <div
      className="carousel-item blue white-text"
      href="#four!"
    >
      <img
        alt="slide3"
        className="slide-img"
        src="../../img/slide3.jpg"
      />
    </div>
  </div>
</div>
`;
